perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -159,18 +159,17 @@ def Prepare(benchmark_spec):
   if vm_util.ShouldRunOnExternalIpAddress():
     # Open all of the command and data ports
     vms[1].AllowPort(PORT_START, PORT_START + num_streams * 2 - 1)
-  netserver_cmd = ('for i in $(seq {port_start} 2 {port_end}); do '
-                   '{netserver_path} -p $i & done').format(
-                       port_start=PORT_START,
-                       port_end=PORT_START + num_streams * 2 - 1,
-                       netserver_path=netperf.NETSERVER_PATH)
+
+  PORT_END = PORT_START + num_streams * 2 - 1
+  netserver_cmd = (f'for i in $(seq {PORT_START} 2 {PORT_END}); do '
+                   f'{netperf.NETSERVER_PATH} -p $i & done')
   vms[1].RemoteCommand(netserver_cmd)
 
   # Copy remote test script to client
   path = data.ResourcePath(os.path.join(REMOTE_SCRIPTS_DIR, REMOTE_SCRIPT))
-  logging.info('Uploading %s to %s', path, vms[0])
+  logging.info(f'Uploading {path} to {vms[0]}')
```

Comment:
Logging should remain in the old style "%s" one.  It gives the runtime a chance to not evaluate the string if the logging level is not enabled.  Usually the linter complains about this...